runtime/syntax/sml.vim

```diff
@@ -4,12 +4,18 @@
 " Maintainers:  Markus Mottl            <markus.mottl@gmail.com>
 "               Fabrizio Zeno Cornelli  <zeno@filibusta.crema.unimi.it>
 " URL:          http://www.ocaml.info/vim/syntax/sml.vim
-" Last Change:  2006 Oct 23 - Fixed character highlighting bug (MM)
-"               2002 Jun 02 - Fixed small typo  (MM)
-"               2001 Nov 20 - Fixed small highlighting bug with modules (MM)
+" Last Change:  2019 Oct 01 - Only spell check strings & comments (Chuanwei Foo)
```

Comment:
It is not necessary to credit me for the fix. Since you have done it already, I would prefer to be credited as "Chuan Wei Foo" instead. Thank you.

Reply:
Ok, sorry for the misspelling!